tests/Unit/ApparentHorizons/Test_StrahlkorperInDifferentFrame.cpp

```diff
@@ -42,7 +42,8 @@ void test_strahlkorper_in_different_frame() {
       1.9, 2.9, 1,
       std::array<size_t, 2>{grid_points_each_dimension,
                             grid_points_each_dimension},
-      false, 1.0, radial_partitioning, radial_distribution, ShellWedges::All,
+      false, {{1.0, 2}}, radial_partitioning, radial_distribution,
```

Comment:
(optional) You can also just pass `std::nullopt` instead of `{{1., 2}}` in all those places where you don't actually want to apply an equatorial compression map. Squash this in directly if you choose to do it.